website/static/js/fileViewTreebeard.js

```diff
@@ -63,11 +63,13 @@ function FileViewTreebeard(data) {
             this.options.showTotal = Math.floor(containerHeight / this.options.rowHeight) + 1;
             this.redraw();
         },
-        lazyLoadOnLoad: function(tree) {
+        lazyLoadOnLoad: function(tree, event) {
```

Comment:
Why include `event` as a parameter of `lazyLoadOnLoad` and not `ontogglefolder`? When I test this locally, `event` is always `undefined` so the scroll still happens when we don't want it to.